python/trigger_ghebackup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+
+
+def get_pid(name: str) -> list:
```

Comment:
You can use `-> List[int]` to specify that this will return a list only with integers. You can import `List` from `typing`